src/base/bittorrent/session.cpp

```diff
@@ -2014,17 +2008,19 @@ bool Session::addTorrent_impl(CreateTorrentParams params, const MagnetUri &magne
             p.flags &= ~lt::torrent_flags::seed_mode;
 #endif
         }
+    }
 
-        if (params.restored && !params.paused) {
-            // Make sure the torrent will restored in "paused" state
-            // Then we will start it if needed
+    // Make sure the torrent will be initially checked and then paused
+    // to perform some service jobs on it. We will start it if needed.
 #if (LIBTORRENT_VERSION_NUM < 10200)
-            p.flags |= lt::add_torrent_params::flag_stop_when_ready;
+    p.flags |= lt::add_torrent_params::flag_paused;
```

Comment:
seems you can merge this block into the block at line 1978

Reply:
It seems then there will be problems with consistent posting of an appropriate code comments.

Reply:
or you can move this block below the block at line 1978, I just wish them to be close, not separated by an `if()` block.

Reply:
> I just wish them to be close, not separated by an if() block.

Ok.